app/src/main/res/layout/layout_appcoins_app_bar.xml

```diff
@@ -0,0 +1,50 @@
+<com.google.android.material.appbar.AppBarLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/app_bar"
+    android:layout_width="match_parent"
+    android:layout_height="@dimen/app_bar_height"
+    android:theme="@style/AppTheme.AppBarOverlay"
+    >
+
+  <com.google.android.material.appbar.CollapsingToolbarLayout
+      android:id="@+id/toolbar_layout"
+      android:layout_width="match_parent"
+      android:layout_height="match_parent"
+      android:background="@drawable/appbar_background_color"
+      android:fitsSystemWindows="false"
+      app:collapsedTitleGravity="start|center_vertical"
```

Comment:
are the lines 16-19 needed for this layout?

Reply:
done